unrestricted-usb-explainer.md

```diff
@@ -0,0 +1,87 @@
+# Explainer for Unrestricted WebUSB
+
+A proposal to enable trusted web applications to bypass security restrictions
+in the WebUSB API.
+
+<!-- TOC -->
+<!-- /TOC -->
+
+## Introduction
+The original intent of the WebUSB API was to support writing companion
+applications for devices not otherwise supported by the browser or operating
+system. Many USB devices use standard drivers included with the operating system
+and are accessed by applications and websites through higher-level interfaces.
+The interface classes for which sites should be using a higher-level interface
+are placed on the [protected interface class list](https://wicg.github.io/webusb/#has-a-protected-interface-class)
+and cannot be claimed by a site using WebUSB. Similarly, the [device blocklist](https://github.com/WICG/webusb/blob/main/blocklist.txt)
+lists a number of security devices which are not accessible to sites using
+WebUSB.
+
+These security restrictions make WebUSB unsuitable for use cases in which an
+application needs to access arbitrary USB devices. This is common in Virtual
+Desktop Infrastructure (VDI) deployments, where local peripherals need to be
+controlled by drivers and applications running on the remote virtual desktop.
```

Comment:
I might rephrase this to be even more specific that this is basically the only use case for this capability:

```suggestion
These security restrictions make WebUSB unsuitable for the USB redirection
features common in Virtual Desktop Infrastructure (VDI) and remote desktop
applications, where local peripherals need to be controlled by drivers and
applications running on a remote system. While high-level redirection is
sometimes possible these systems typically require low-level USB access in
order to provide the best compatibility.
```